.github/workflows/ci.yml

```diff
@@ -54,3 +54,15 @@ jobs:
 
       - name: Run tests
         run: npm test --ignore-scripts
+
+  smoketest:
```

Comment:
I'm debating when this thing should run. Because it takes a while running it on every PR commit seems like overkill. Maybe it should only run on `main` and we'll (me) will just need to wait to tag releases until it passes there?